WORKSPACE

```diff
@@ -265,3 +265,9 @@ load("@vaticle_bazel_distribution//common:rules.bzl", "workspace_refs")
 workspace_refs(
     name = "vaticle_typedb_driver_workspace_refs"
 )
+
+new_local_repository(
+    name = "gherkin_cpp",
+    path = "/Users/krishnangovindraj/code/dependency_libraries/dist/gherkin",
+    build_file = "cpp/test/cucumber/gherkin.bzl"
+)
```

Comment:
I should already upload this somewhere so I don't lose it if my disk dies.

Reply:
Do we build from source (with the build file contents specified in the http_archive target)?

Gherkin has dependencies (and transitive dependencies, on the json library), which might make it annoying.
The easiest solution is to upload the binaries & headers (i.e. the contents of the folder here) for just the platforms we want and keep this approach.